spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/restart/RestartApplicationListener.java

```diff
@@ -81,15 +85,6 @@ private void onApplicationFailedEvent(ApplicationFailedEvent event) {
 
 	@Override
 	public int getOrder() {
-		return this.order;
-	}
-
-	/**
-	 * Set the order of the listener.
-	 * @param order the order of the listener
-	 */
-	public void setOrder(int order) {
-		this.order = order;
+		return HIGHEST_PRECEDENCE;
```

Comment:
I have no opinion has whether this should be changed but we should have this discussion in a separate PR.

Reply:
changed it back